### DISCUSSION TITLE
Blank screen after step 4

### DISCUSSION THREAD

**Discussion by Tomvtzand** [PodcastGenerator/PodcastGenerator#754] [General]
Hi all,

I'm trying to install podcast generator on my ubuntu-server and at first it looks good, but after step 4 of the installation step 5 turns up as a blank screen.

I've tried changing php versions, permissions and ownership settings, but nothing works. 
Does anyone of you have any solutions?

My setup:
Ubuntu 22.04
php8.1
apache2

Thanks, 
Tom

**Comment by CaptainPackers** [comment]
> Hi all,
> 
> I'm trying to install podcast generator on my ubuntu-server and at first it looks good, but after step 4 of the installation step 5 turns up as a blank screen.
> 
> I've tried changing php versions, permissions and ownership settings, but nothing works. 
> Does anyone of you have any solutions?
> 
> My setup:
> Ubuntu 22.04
> php8.1
> apache2
> 
> Thanks, 
> Tom

I just unzipped the package in a folder in my public html folder and it worked fine. I retained the pg folder so my url is myserverurl/pg

-- Captain

**Comment by richtestani** [comment]
The blank screen is because php errors are turned off. 
It's likely the one of the config files is not being found. Turn on your errors and you will see the issue.
